content/redirect.js

```diff
@@ -1,26 +1,11 @@
 const fs = require("fs");
 const path = require("path");
+const { textChangeRangeIsUnchanged } = require("typescript");
 
 const { CONTENT_ROOT, VALID_LOCALES } = require("./constants");
+const { resolveFundamental } = require("./fundamental-redirects");
 const { buildURL } = require("./utils");
 
-function add(locale, oldSlug, newSlug) {
```

Comment:
I deleted it because it looked like it wasn't used. There's a reference to `this.allRedirects` which is `undefined` and `contentRoot` probably should be `CONTENT_ROOT`. I'll rewrite it and add it again.